File: hello.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  hello.py
#  
#  Copyright 2017  <>

ROK_TERAZ = 2017
ROK_PY = 1991
def parzyste(n):
    i = list(range(0 , n+1, 2))
    print (i, " ")
    print(ile)
    return len(ile)

def main(args):
    
    imie =  input("Podaj imię: ")
    
    print("Witam ", imie)
    wiek =  int(input("Podaj wiek: "))
    print("Wiek: ", wiek)
    rok=ROK_TERAZ-wiek
    if(rok>ROK_PY):
        print("Jesteś młodszy od języka PY")
    elif(rok<ROK_PY):
        print("Jesteś starszy od języka PY")
    elif(rok==ROK_PY):
        print("Mamy tyle samo lat")
    
    n = int(input("Podaj liczbę :"))
    print ("Parzyste: ", parzyste(n))
        
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: kody/hello.cpp

#include <iostream>
using namespace std;

#define ROK_TERAZ 2017
#define ROK_CPP 1983
void parzyste(int a)
{
    cout<<"Parzyste: "<<endl;
    for (int i=0; i<a; i=i+2)
    cout <<i<<" ";
    
}
int main(int argc, char **argv)
{
    string imie;
    int wiek;
    int rok;
    int numer;
	cout<<"Czesc, podaj imię"<<endl;
    cin>>imie;
    cout << "Witaj "<< endl << imie << endl;
    cin>>wiek;
    cout << "Rok: "<< ROK_TERAZ - wiek << endl;
    rok = ROK_TERAZ - wiek;
    if (rok> ROK_CPP)
    {
        cout<<"Jesteś młodszy od języka CPP"<<endl;
    }
        else if (rok<ROK_CPP){
        
        cout<<"Jesteś starszy od języka CPP"<<endl;
    }
    cout<<"Podaj liczbę: "<<endl;
    
    cin>>numer;
    parzyste(numer);
    
	return 0;
}

